### DISCUSSION TITLE
osquery alerts and rules

### DISCUSSION THREAD

**Discussion by aremai** [Security-Onion-Solutions/securityonion#3104] [Unsupported Versions] [answered]
Hello there,

I've studied the security onion documentation [1] thoroughly, but I can't seem to find a way how to trigger alerts with osquery. Are alerts just used for Suricata, Wazuh and Sigma?
There also doesn't seem to be a way to configure local rules for osquery itself..... I know that this probably obsolete since the queries are configured within Fleet. 

In order to have alerts triggered would I first need to define local rules in Wazuh to pick up events coming from osquery? 
What is the recommended procedure here? 

Also, I'm confused, where would I find the wazuh ruleset? It isn't located in /var/ossec/ruleset/ and it's also neither in those locations described here? https://docs.securityonion.net/en/2.3/rules.html#rulesets
In fact this guide does not describe how to configure Wazuh/ossec rules at all. So, how would you configure those in order to invoke new alerts?

Looking forward to hearing from you.

thanks,
theresa

[1] https://docs.securityonion.net/en/2.3/managing-alerts.html

**Comment by defensivedepth** [comment, marked as answer]
Osquery/Fleet do not have an alerting engine builtin - the typical way to generate alerts for osquery data is by using Playbook. (https://docs.securityonion.net/en/2.3/playbook.html)

You can load an example osquery Play by navigating to "Create New Play" one you open Playbook.
